chainermn/__init__.py

```diff
@@ -7,6 +7,7 @@
 from chainermn import iterators  # NOQA
 from chainermn import links  # NOQA
 from chainermn import optimizers  # NOQA
+import chainermn.util  # NOQA
```

Comment:
This case so specific that the file name should be specific like global-hook or global-exception-hook, or something like that.

Reply:
I rename the file to `global_exc_hook.py`.